src/cascadia/TerminalSettingsEditor/Profiles.xaml

```diff
@@ -66,7 +66,8 @@
                             so the name will always be overridden.
                         -->
                         <local:SettingContainer x:Uid="Profile_Name"
-                                                Margin="0,0,0,24"
+                                                CurrentValue="{x:Bind Profile.Name, Mode=OneWay}"

```

Comment:
Why do only these 5 have `CurrentValue` set?

Reply:
There are 2 styles of 'expanders': one which doesn't actually expand and has its content on the right side of the header (like the default profile setting), and one which does expand and shows its current value on the right side of the header (like profile name). Only the latter expander style cares about current value